### ISSUE TITLE
Global install failed - Node 0.12.5 with nvm on OS X

### ISSUE THREAD

**Issue by ericelliott** [azproduction/node-mc#2]
Node 0.12.5 with nvm on OS X.

``` js
23869 verbose rebuildBundles   'socket.io-client',
23869 verbose rebuildBundles   'source-map-support',
23869 verbose rebuildBundles   'stats.js',
23869 verbose rebuildBundles   'superagent',
23869 verbose rebuildBundles   'term-mouse',
23869 verbose rebuildBundles   'then-jade' ]
23870 verbose unlock done using /Users/eric/.npm/_locks/node-mc-a1a22623561ca513.lock for /Users/eric/.nvm/versions/node/v0.12.5/lib/node_modules/node-mc
23871 verbose stack Error: ENOENT, chmod '/Users/eric/.nvm/versions/node/v0.12.5/lib/node_modules/node-mc/bin/nmc'
23871 verbose stack     at Error (native)
23872 verbose cwd /Users/eric/Dropbox/dev/cuid
23873 error Darwin 14.1.0
23874 error argv "/Users/eric/.nvm/versions/node/v0.12.5/bin/node" "/Users/eric/.nvm/versions/node/v0.12.5/bin/npm" "i" "node-mc" "-g"
23875 error node v0.12.5
23876 error npm  v2.11.2
23877 error path /Users/eric/.nvm/versions/node/v0.12.5/lib/node_modules/node-mc/bin/nmc
23878 error code ENOENT
23879 error errno -2
23880 error enoent ENOENT, chmod '/Users/eric/.nvm/versions/node/v0.12.5/lib/node_modules/node-mc/bin/nmc'
23880 error enoent This is most likely not a problem with npm itself
23880 error enoent and is related to npm not being able to find a file.
```

**Comment by azproduction** [comment]
Thanks for reporting. Is it working without -g? I'll fix it once I have decent internet connection.

**Comment by SebastianZaha** [comment]
Can confirm, I'm getting the same error. Also happening without -g.

Log: https://www.dropbox.com/s/4thdwbxikrmdk85/npm-debug.log?dl=1

**Comment by fetchak** [comment]
Confirmed here too, Node 0.12.5 on Linux.

**Comment by azproduction** [comment]
Fixed in `node-mc@0.2.1`. 

Due to PhantomJS font issues you wont be able to have full experience with nmc. Please run it in development mode.

```
git clone git://github.com/azproduction/node-mc.git
cd node-mc
npm install
npm run debug
```
